danPortfolio.css

```diff
@@ -0,0 +1,117 @@
+@import url('https://fonts.googleapis.com/css2?family=Itim&display=swap');
+
+@font-face {
+    font-family: "Itim";
+    src: url("../fonts/Itim-Regular.ttf") format('truetype');
+}
+
+* {
+	font-family: 'Itim';
+/*	overflow: auto;*/
+	box-sizing: border-box;
+}
+
+body {
+/*	display: flex;*/
+	align-self: flex-start;
+	
+}
+
+p, h2, h3, h4 {
+	margin: 0;
+}
+
+h2 {
+	padding: 10px 0px;
+}
+
+nav {
+    position: fixed;
+    width: 100%;
+    top: 0;
+    left: 0;
+    background: #3197d6;
+    padding: 15px;
+    display: flex;
+    justify-content: space-between;
+    z-index: 2;
+}
+
+a {
+	color: black;
+}
+
+.box {
+	max-width: 50%;
+}
+
+a:hover { 
+  color: red;
+}
+
+.download-cv {
+	border-radius: 25px;
+	border: solid red;
+}
+
+.intro {
+	background-color: #3197d6;
+	margin: 0px;
+	display: flex;
+	justify-content: space-evenly;
+	height: 100%;
+	padding: 75px;
+}
+
+.brief-intro {
+	padding: 25px;
+}
+
+.about {
```

Comment:
adding flex-wrap: wrap will allow you to move picture from side to top/underneath